Program.fs

```diff
@@ -689,24 +708,21 @@ let rec private runner (args : FemtoArgs) =
             | Error er ->
                 logger.Error("Error while analyzing the project's structure and dependencies")
                 FemtoResult.ValidationFailed
-            | Ok _ ->
-                let libraryName = Path.GetFileNameWithoutExtension project
-                let npmDependencies = Npm.parseDependencies project
-                if List.isEmpty npmDependencies then
-                    logger.Warning("Project {Project} does not contain npm dependency metadata", libraryName)
-                    FemtoResult.ProjectCrackerFailed
-                else
-                    for pkg in npmDependencies do
-                        logger.Information("{Library} requires npm package {Package}", libraryName, pkg.Name)
-                        logger.Information("  | -- Required range {Range}", pkg.RawVersion)
-                        logger.Information("  | -- Resolution strategy '{Strategy}'", if pkg.LowestMatching then "Min" else "Max")
-                        match getSatisfyingPackageVersion NodeManager.Npm pkg with
-                        | Some version ->
-                            logger.Information("  | -- √ Found version {Version} that satisfies the required range", version)
-                        | None ->
-                            logger.Error("  | -- Could not find a version that satisfies the required range {Range}", pkg.RawVersion)
+            | Ok crackedProject ->
+                let libraries = findLibraryWithNpmDeps crackedProject
 
+                let isValid =
```

Comment:
Can you please simplify `isValid` into this
```fs
let isValid = libraries |> List.forall validateProject 
```

Reply:
Sure will due, I didn't know about this function :)

Reply:
Finally, we can't replace it.

`List.forall` will stop on the first error.

Reply:
I see, my bad I guess `List.forall` is lazy. Alright then, feel free to merge if you are finished with the PR ;)